.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  build_and_test:
+    name: Build and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10, 12]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node }}
+    - run: yarn install
+    - run: yarn build
+    - run: yarn test
```

Comment:
we could also run `yarn lint`

Reply:
yes to lint, but these should be separate jobs